q2_diversity/_alpha/_visualizer.py

```diff
@@ -264,18 +264,21 @@ def _compute_rarefaction_data(feature_table, min_depth, max_depth, steps,
 
 
 def alpha_rarefaction(output_dir: str, table: biom.Table, max_depth: int,
-                      phylogeny: skbio.TreeNode=None, metric: str=None,
+                      phylogeny: skbio.TreeNode=None, metrics: set=None,
                       metadata: qiime2.Metadata=None, min_depth: int=1,
                       steps: int=10, iterations: int=10) -> None:
-    if metric is None:
+
+    if metrics is None:
         metrics = ['observed_otus', 'shannon']
```

Comment:
Maybe turn these into `{'observed_otus', 'shannon'}` and use `metrics.add('faith_pd')`

Reply:
That way downstream code can expect the type to always be a set

Reply:
Good call 👍